frontend-nanodegree-arcade-game/index1.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Effective JavaScript: Frogger</title>
+    <link rel="stylesheet" href="css/style.css">
+    <script
+  src="https://code.jquery.com/jquery-1.12.4.min.js"
+  integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="
+  crossorigin="anonymous"></script>
+</head>
+<body>
+    <script type="text/javascript" src="js/resources.js"></script>
+    <script type="text/javascript" src="js/app.js"></script>
+    <script type="text/javascript" src="js/engine.js"></script>
+    <script type="text/javascript" src="js/idb.js"></script>
+    <script type="text/javascript" src="js/main.js"></script>
+    <script type="text/javascript" src="js/popup.js"></script>
+
+    <div class="scores">
+
+        <h1 id="score" class="score"></h1>
+
+        <h1 id="topScore" class="topScore"></h1>
+
+        <h4 class="about"> <a class="trigger_popup_fricc">About the game</a>
+
+            <div class="hover_bkgr_fricc">
+                <span class="helper"></span>
+                <div>
+                    <div class="popupCloseButton">X</div>
+                        
+                        
+                        <h1> About the game !</h1>
```

Comment:
spacing after the h1 tag:

 <h1> About the game !</h1>

Maybe it should be:

<h1>About the game !</h1>